SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorModule.cpp

```diff
@@ -80,6 +84,32 @@ FString FSpatialGDKEditorModule::GetSpatialOSCloudDeploymentName() const
 	return GetDefault<USpatialGDKEditorSettings>()->DevelopmentDeploymentToConnect;
 }
 
+bool FSpatialGDKEditorModule::ShouldConnectServerToCloud() const
+{
+	return GetDefault<USpatialGDKEditorSettings>()->IsConnectServerToCloudEnabled();
+}
+
+bool FSpatialGDKEditorModule::TryStartLocalReceptionistProxyServer() const
+{
+	if (ShouldConnectToCloudDeployment() && ShouldConnectServerToCloud())
+	{
+		bool bSuccess = LocalReceptionistProxyServerManager->TryStartReceptionistProxyServer(GetDefault<USpatialGDKSettings>()->IsRunningInChina(), GetDefault<USpatialGDKEditorSettings>()->DevelopmentDeploymentToConnect, GetDefault<USpatialGDKEditorSettings>()->ListeningAddress, GetDefault<USpatialGDKEditorSettings>()->LocalReceptionistPort);
+
+		if (bSuccess)
+		{
+			UE_LOG(LogTemp, Log, TEXT("%s"), *LOCTEXT("ProxyStartedSuccessfully", "Successfully started local receptionist proxy server!").ToString());
```

Comment:
The console might not support localized text, so logs can't really be localized. It is generally accepted that this is the case (and Unreal native logs aren't localized either)
```suggestion
			UE_LOG(LogTemp, Log, TEXT("Successfully started local receptionist proxy server!"));
```

Reply:
ah, I'm sorry about that. Think I was the one who told you to look into localized text. So do we need to do localized text only for dialog boxes?

Reply:
Yes, anything that appears in the UI or message boxes can benefit from being localized. Localized text could potentially work in logs as well but I can only see 3 occurrences of that in the engine code, and I don't know for a fact that they actually have the localized versions, so I'm a little concerned that we may run into issues with the console displaying garbage strings if it doesn't support Chinese characters.